packages/desktopjs-electron/README.md

```diff
@@ -0,0 +1,3 @@
+# @morgan-stanley/desktopjs-electron
```

Comment:
This is mostly a placeholder readme.  I'm debating whether I'll have info here or just link to the main project as-is.